framework/math/PETScUtils/petsc_utils_04_vecmanip.cc

```diff
@@ -58,31 +82,31 @@ void chi_math::PETScUtils::CopyGlobalVecToSTLvector(
 
   //=================================== Creating PETSc vector
   Vec local_vec;
-  VecCreateSeq(PETSC_COMM_SELF,global_indices.size()+1,&local_vec);
-  VecSet(local_vec,0.0);
+  VecCreateSeq(PETSC_COMM_SELF, global_indices.size() + 1, &local_vec);
+  VecSet(local_vec, 0.0);
 
   //=================================== Create and transfer index sets
   IS global_set;
   IS local_set;
-  ISCreateGeneral(PETSC_COMM_SELF, N, global_indices.data(),
-                  PETSC_COPY_VALUES,&global_set);
-  ISCreateGeneral(PETSC_COMM_SELF, N, local_indices.data(),
-                  PETSC_COPY_VALUES,&local_set);
+  ISCreateGeneral(
+    PETSC_COMM_SELF, N, global_indices.data(), PETSC_COPY_VALUES, &global_set);
+  ISCreateGeneral(
+    PETSC_COMM_SELF, N, local_indices.data(), PETSC_COPY_VALUES, &local_set);
   VecScatter scat;
-  VecScatterCreate(x,global_set,local_vec,local_set,&scat);
-  VecScatterBegin(scat,x,local_vec,INSERT_VALUES,SCATTER_FORWARD);
-  VecScatterEnd(scat,x,local_vec,INSERT_VALUES,SCATTER_FORWARD);
+  VecScatterCreate(x, global_set, local_vec, local_set, &scat);
+  VecScatterBegin(scat, x, local_vec, INSERT_VALUES, SCATTER_FORWARD);
+  VecScatterEnd(scat, x, local_vec, INSERT_VALUES, SCATTER_FORWARD);
 
   //=================================== Copy to STL
   data.clear();
-  data.resize(N,0.0);
+  data.resize(N, 0.0);
   const double* x_ref;
-  VecGetArrayRead(local_vec,&x_ref);
+  VecGetArrayRead(local_vec, &x_ref);
 
-  for (size_t i=0; i<N; ++i)
+  for (size_t i = 0; i < N; ++i)
```

Comment:
Another copy. Definitely a fan of stl where we can use it.

Reply:
Done